sdk/nodejs/yarn.lock

```diff
@@ -2645,6 +2675,18 @@ rimraf@^3.0.0, rimraf@^3.0.2:
   dependencies:
     glob "^7.1.3"
 
+rome@^12.0.0:
```

Comment:
Shouldn't there be an accompanying package.json change?

Reply:
Yes, [it's here](https://github.com/pulumi/pulumi/pull/12789/commits/2e12a4cb4b04366c635483a488409335f0b1e09d#diff-0d53ce0d334b84822a2dbdf673d0456efbacb59a76f72aabbfb1b7f8203a58f7R55).